### ISSUE TITLE
Incompatible with coq 8.15.2?

### ISSUE THREAD

**Issue by SnarkBoojum** [tchajed/coq-record-update#36]
I'm trying to package for Debian, and the compilation fails at the testing phase:
```
COQTEST tests/RegressionTests.v (ref: tests/RegressionTests.ref)
--- tests/RegressionTests.ref   2021-05-27 16:49:14.000000000 +0200
+++ /tmp/tmp.PRSIEUHhZM 2022-07-16 12:00:28.576987924 +0200
@@ -1,3 +1,4 @@
+File "/home/jpuydt/Debian/build/toto/coq-record-update-0.3.0/tests/RegressionTests.v", line 19, characters 2-66:
 The command has indeed failed with message:
 The following term contains unresolved implicit arguments:
   (fun (r : X) (a : nat) => r <| getA := a |>)
@@ -9,3 +10,5 @@
 test = 
 fun (s : ThreadState) (newRegs : Registers) => s <| Regs := newRegs |>
      : ThreadState -> Registers -> ThreadState
+
+Arguments test s newRegs
```

**Comment by tchajed** [comment]
Thanks for the report! It seems Coq output has changed since the last release so the tests fail on v0.3.0 even though they work on master. I've just released v0.3.1 which should fix this issue.

**Comment by SnarkBoojum** [comment]
Perfect, thanks!
